src/main/java/org/auscope/portal/server/web/controllers/RockPropertiesController.java

```diff
@@ -0,0 +1,54 @@
+package org.auscope.portal.server.web.controllers;
+
+import org.auscope.portal.core.server.OgcServiceProviderType;
+import org.auscope.portal.core.server.controllers.BasePortalController;
+import org.auscope.portal.core.services.methodmakers.filter.FilterBoundingBox;
+import org.auscope.portal.core.services.responses.wfs.WFSTransformedResponse;
+import org.auscope.portal.server.web.service.RockPropertiesService;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.stereotype.Controller;
+import org.springframework.web.bind.annotation.RequestMapping;
+import org.springframework.web.bind.annotation.RequestParam;
+import org.springframework.web.servlet.ModelAndView;
+
+@Controller
+public class RockPropertiesController extends BasePortalController {
+	
+	private RockPropertiesService rockPropertiesService;
+
+    // ----------------------------------------------------------- Constructors
+
+	private static final String SERVICE_URL = "http://www.ga.gov.au/geophysics-rockpropertypub-gws/ga_rock_properties_wfs/wfs";

```

Comment:
Not sure about this endpoint being hardcoded, isn't that configurable in known layers or something?